Chrismarcel/iReporter
# #163422496 Implement user dashboard

__What does this PR do?__
This PR implements the user dashboard which allows an authenticated user view their reports, reports statistics and also edit and delete their reports.

__Description of tasks__
- Ensure that authenticated user can view their dashboards
- Ensure that users posts are retrieved from the database and displayed on the dashboard

__How to test this PR__
- Visit https://chrismarcel.github.io/iReporter/UI/login.html
- Log in with your username and password
- Visit the https://chrismarcel.github.io/iReporter/UI/dashboard.html
- You should be able to see all the reports you have created

__Link to relevant stories__
[#163422496](https://www.pivotaltracker.com/story/show/163422496)
